vagrantfiles/config.yaml

```diff
@@ -0,0 +1,13 @@
+vagrantfile:
+    synced_folder_guest: '/home/vagrant/imwatchingyou'
+    ruby_version: 2.0.0
+    vm:
+        box: 'debian/jessie64'
+        cpus: '1'
+        memory: '1024'
+        network:
+            private_network: 192.168.12.34
+            forwarded_port:
+                ports:
+                    host: '8080'
```

Comment:
I'll leave the `3030` port for the moment since we can't use `80` unless `vagrant` is executed using _root_ priliveges.